docs/source/conf.py

```diff
@@ -37,7 +38,7 @@
     "sphinx.ext.viewcode",
     "sphinxcontrib.jquery",
     "nbsphinx",
-    "recommonmark",
+    "myst_parser",
```

Comment:
recommonmark is no longer maintained, `myst_parser` is the recommended alternative.